.github/workflows/test.yml

```diff
@@ -18,3 +18,7 @@ jobs:
         with:
           dotnet-version: "7.0.100"
       - run: pwsh ./test.ps1
+      - uses: test-summary/action@v2
```

Comment:
Could you pin this to a SHA? See https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas